spirv_glsl.hpp

```diff
@@ -90,6 +90,13 @@ class CompilerGLSL : public Compiler
 		// If disabled on older targets, binding decorations will be stripped.
 		bool enable_420pack_extension = true;
 
+		// If true, all two-vector OpVectorShuffles will be forced to use a temporary variable.
```

Comment:
This is a bit awkward to make into an option, this should just be forced.

Reply:
Always forcing makes sense for MSL. But maybe not GLSL and HLSL.

I've moved it to a backend config selection.